pkg/lib/repo/remote/upload-format_test.go

```diff
@@ -101,3 +101,30 @@ func TestGetUploadFormatGoogleArtifactRegistry(t *testing.T) {
 		assert.Equal(t, uploadMonolithicPut, uploadFormatLarge, "Large layers should use monolithic put")
 	}
 }
+
+func TestGetUploadFormatAmazonECR(t *testing.T) {
+	testRegistries := []string{
```

Comment:
We should add a few cases with mixed case URLs.

Reply:
Good hint -- we're not handling them at all right now :slightly_smiling_face: